share/p2p/peers/pool.go

```diff
@@ -54,6 +54,11 @@ func (p *pool) tryGet() (peer.ID, bool) {
 		return "", false
 	}
 
+	// if pointer is out of range, point to first element
+	if p.nextIdx > len(p.peersList)-1 {
```

Comment:
this always assumes there's at least 1 peer in peerslist yeah? is it possible that there are none -- meaning activeCount > 0 but peerslist ==0

Reply:
It should be impossible